blog/blog.html

```diff
@@ -1,52 +1,22 @@
 <!DOCTYPE html>
-
-<html class="no-js" lang="en">
+<html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="description" content="" />
-    <meta name="author" content="" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, maximum-scale=1"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
+
+    <title>KyootyBellas blog</title>
 
-    <script src="js/modernizr.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/validate.js/0.13.1/validate.min.js"></script>
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
 
-    <script src="../scripts/index.js"></script>
-    <title>KyootyBellas blog</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
     <link rel="stylesheet" href="../css/global/blog.css" />
+    <link rel="icon" href="../assets/logos/favicon.ico" type="image/png" />
+    <link rel="apple-touch-icon" sizes="180x180" href="../assets/logos/apple-touch-icon.png"/>
+    <link rel="icon" type="image/png" sizes="32x32" href="../assets/logos/favicon-32x32.png"/>
+    <link rel="icon" type="image/png" sizes="16x16" href="../assets/logos/favicon-16x16.png"/>
+    <link rel="mask-icon" href="../assets/logos/safari-pinned-tab.svg" color="#5bbad5"/>
 
-    <link rel="icon" href="../photos/icons/favicon.ico" type="image/png" />
-    <link
-      rel="apple-touch-icon"
-      sizes="180x180"
-      href="../photos/icons/apple-touch-icon.png"
-    />
-    <link
-      rel="icon"
-      type="image/png"
-      sizes="32x32"
-      href="../photos/icons/favicon-32x32.png"
-    />
-    <link
-      rel="icon"
-      type="image/png"
-      sizes="16x16"
-      href="../photos/icons/favicon-16x16.png"
-    />
-    <link rel="manifest" href="photos/icons/site.webmanifest" />
-    <link
-      rel="mask-icon"
-      href="../photos/icons/safari-pinned-tab.svg"
-      color="#5bbad5"
-    />
```

Comment:
It's a bit of a personal preference thing perhaps, but I'd say you really don't want to space these out in the head of your HTML because they are not relevant to constantly look at ( they don't need to be "readable" in that sense ). You just want to quickly get to the relevant part, which is your template.

Reply:
yeah, this is just prettier doing prettier things, I don't know why it spaces out like that :D